attesters/tsm/tsm.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package tsm
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/google/go-configfs-tsm/configfs/linuxtsm"
+	"github.com/google/go-configfs-tsm/report"
+	"github.com/veraison/cmw"
+	"github.com/veraison/ratsd/api"
+	"github.com/veraison/ratsd/proto/compositor"
+)
+
+const (
+	ApplicationvndVeraisonConfigfsTsmCbor = "application/vnd.veraison.configfs-tsm+cbor"
+)
+
+var (
+	sid = &compositor.SubAttesterID{
+		Name:    "tsm-report",
+		Version: "1.0.0",
+	}
+
+	supportedFormats = []*compositor.Format{
+		&compositor.Format{
+			ContentType: string(api.ApplicationvndVeraisonConfigfsTsmJson),
+			NonceSize:   32,
+		},
+		&compositor.Format{
+			ContentType: ApplicationvndVeraisonConfigfsTsmCbor,
+			NonceSize:   32,
+		},
+	}
+
+	statusSucceeded = &compositor.Status{Result: false, Error: ""}
```

Comment:
similar to https://github.com/veraison/ratsd/pull/35#discussion_r1981123226, on success status.result = true.

It’s probably worth a comment in compositor.proto :-)

Reply:
Added another commit to add the comment.